src/entities/hrly-op-data.entity.ts

```diff
@@ -151,42 +151,40 @@ export class HrlyOpData extends BaseEntity {
     c => c.hrlyOpData,
   )
   @JoinColumn({ name: 'hour_id' })
-  derivedHrlyValues: DerivedHrlyValue[];
+  derivedHourlyValue: DerivedHrlyValue[];
```

Comment:
For the other entities, the pattern followed for the names has been the entity name and making it plural.
To be consistent it should be: `derivedHrlyValues: DerivedHrlyValue[];`

Similar changes needed to be made to the other relationships below